### PR TITLE
ResearchGate Field

### PR DESCRIPTION
**Pull request by kch8qx**
Make changes to ResearchGate field to reflect the need to add user's RG logo/link when displaying search results, and add RG as a user attribute.

[OSF-2427]

### CODE REVIEW

File: website/templates/search.mako [hunk 1]
```diff
@@ -169,6 +169,7 @@
                     <span data-bind="visible: school, text: school"></span><!-- ko if: degree || school --><br /><!-- /ko -->
                 </p>
                 <!-- ko if: social -->
+                <!-- <p>social.researchGate</p> -->
```
Comment (mfraezz) [inline website/templates/search.mako]: Is this necessary?

Comment (kch8qx) [inline website/templates/search.mako]: whoops, nope that was left in accidentally

**Comment by mfraezz** [comment]
LGTM :shipit: 

Pending the removal of an HTML comment, I'll send this on to RtM

**Comment by sloria** [comment]
Merged into develop with the above minor change.

**Pull request closed without merging**